src/v2/lang/en-us.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Wondering what other's think, but should we keep all per-language translations in one single file? I can imagine it getting messy, but at the same time I can also imagine it being functionally easier to use add to and revise, all things considered. 

Another option would be to follow the pattern that we've got setup for tests and which everyone is familiar with and add folder-level `__i18n__`. So like:

```bash
├── Bar.tsx
├── Foo.tsx
├── __i18n__
│   ├── Bar.i18n.json
│   └── Foo.i18n.json
└── __tests__
    ├── Bar.test.tsx
    └── Foo.test.tsx
```